python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -731,13 +746,13 @@ def _actually_copyFrom(self, _srcobj, _ignore_atts):
             if not self._schema.hasAttribute(name):
                 #raise ValueError('copyFrom: incompatible schema: source=%s destination=%s'%(_getName(_srcobj), _getName(self)))
                 if not hasattr(self, name):
-                    setattr(self, name, self._schema.getDefaultValue(name))
+                    setattr(self, name, self._schema.getDefaultValue(name, make_copy=True))
```

Comment:
No but it's useful to see that there is a copy being performed in this call to explain why it's so heavy.